doc/api/wasi.md

```diff
@@ -148,6 +146,19 @@ added:
     WebAssembly application. **Default:** `1`.
   * `stderr` {integer} The file descriptor used as standard error in the
     WebAssembly application. **Default:** `2`.
+  * `version` {string} The version of wasi requested. Currently the only
+    supported version is `wasi_snapshot_preview1`
+
+### `wasi.getImportObject()`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Return an import object that can be passed to `wasi.start` if
```

Comment:
The result of this function isn't supposed to be passed to `wasi.start()` is it?

Reply:
Thanks for catching that.

Reply:
fixed